### PR TITLE
[ET-1078] Blocks > Fix RSVP & Tickets block description

### PR DESCRIPTION
**Pull request by juanfra**
### 🎫 Ticket

[ET-1078] 

### 🗒️ Description

- Switching the tickets and RSVP blocks to have an autosize textarea so it doesn't break the block when using long descriptions.
- Few aesthetics amendments (i.e.: using the design system black)

### 🎥 Artifacts 

https://www.loom.com/share/6c943f7a82034284b134eef49551054a

### ✔️ Checklist
- [x] I've included a readme.txt Changelog entry. 
- [x] My code is tested. 
- [x] My code has [proper inline documentation](https://the-events-calendar.github.io/products-engineering/docs/code-standards/).

[ET-1078]: https://theeventscalendar.atlassian.net/browse/ET-1078

### CODE REVIEW

File: src/modules/blocks/rsvp/container-header/style.pcss [hunk 1]
```diff
@@ -94,39 +94,40 @@
...
+		font-size: 12px;
+		line-height: 18px;
+		letter-spacing: 0.04px;
+		max-height: 108px;
 
-		& > * {
-			flex: none;
-		}
+		font-size: 12px;
+		line-height: 1.38;
```
Comment (paulmskim) [inline src/modules/blocks/rsvp/container-header/style.pcss:111]: got an empty line here. also double declaration of font size and line height?

File: src/modules/blocks/ticket/container-header/description/style.pcss [hunk 2]
```diff
@@ -16,26 +16,29 @@
...
-		& > input {
-			background-color: transparent;
-			color: #545D66;
-			margin: 0;
-			padding: 0;
-			border: none;
-			font-size: 12px;
-			line-height: 18px;
-			letter-spacing: 0.04px;
+		&:focus {
```
Comment (paulmskim) [inline src/modules/blocks/ticket/container-header/description/style.pcss:35]: does this need a hover style too?

**Review by paulmskim — CHANGES_REQUESTED** [review]
A couple small things. Also, it's probably fine, but could you add an artifact of what an unfocused multiline description looks like?

**Comment by juanfra** [comment]
Thanks @paulmskim - Submitted the changes.

The artifact for that can be found here: https://www.loom.com/share/c3c3fb6a9cf5484c9cdbdc14f2ae7ff6

Please let me know if there's something else missing.

File: readme.txt [hunk 3]
```diff
@@ -178,7 +178,7 @@ Check out our extensive [knowledgebase](https://evnt.is/18wm) for articles on us
 
 = [5.1.3] TBD =
 
-
+* Fix - Fixed an issue where long descriptions were breaking the block for the Tickets and RSVP blocks. It's now using auto-size textarea. [ET-1078]
```
Comment (sc0ttkclark) [inline readme.txt:181]:
```suggestion
* Fix - Fixed an issue with Tickets and RSVP blocks where long descriptions were breaking the block. They now use an auto-resizing textarea. [ET-1078]
```

**Review by sc0ttkclark — APPROVED** [review]

**Comment by paulmskim** [comment]
@juanfra awesome, yeah that was exactly what i was looking for, when the rsvp/ticket wasn't in edit state and showed the condensed view. Thanks!

**Review by paulmskim — APPROVED** [review]

**Pull request merged**